hw_diag/tests/test_views_diagnostics.py

```diff
@@ -69,7 +69,10 @@ def test_initFile_output(self):
             self.assertEqual(resp.status_code, 200)
             self.assertEqual(resp.status, '200 OK')
 
-    @patch.dict(os.environ, {'FIRMWARE_VERSION': '1337.13.37', 'DIAGNOSTICS_VERSION': 'aabbffe'})
+    @patch.dict(
+        os.environ,
+        {'FIRMWARE_VERSION': '1337.13.37', 'DIAGNOSTICS_VERSION': 'aabbffe'}
```

Comment:
![hack the planet](https://media.giphy.com/media/FnGJfc18tDDHy/source.gif)

Reply:
@vpetersson - of course you would notice Mr. `mvip@hax0rpad3` :smiley: